src/appmixer/quickbooks/accounting/NewCustomer/NewCustomer.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { makeRequest } = require('../../commons');
+
+module.exports = {
+
+    async tick(context) {
+
+        let { changedSince } = await context.stateGet('changedSince') || {};
+
+        if (!changedSince) {
+            // On first tick, fetch only the most recent update to set changedSince
+            changedSince = new Date();
+            await context.stateSet('changedSince', { changedSince });
+        }
+
+        const options = {
+            path: `v3/company/${context.profileInfo.companyId}/cdc?entities=Customer&changedSince=${changedSince.toISOString()}`,
+            method: 'GET'
+        };
+
+        try {
+            const { data } = await makeRequest({ context, options });
+
+            // Empty response: {"CDCResponse": [{"QueryResponse": [{}]}],"time": "2024-04-18T05:12:07.655-07:00"}
+            if (data.CDCResponse[0].QueryResponse.length > 0 && data.CDCResponse[0].QueryResponse[0].Customer) {
```

Comment:
unnecessary repetition of the code segment `data.CDCResponse[0].QueryResponse[0]`.